core/gui/package.json

```diff
@@ -55,8 +55,10 @@
     "jszip": "3.10.1",
     "lodash-es": "4.17.21",
     "marked": "^4.3.0",
-    "monaco-editor": "0.36.1",
-    "monaco-languageclient": "5.0.1",
+    "monaco-editor": "0.51.0",
+    "monaco-languageclient": "8.8.3",
+    "vscode": "npm:@codingame/monaco-vscode-api@~8.0.4",
+    "vscode-ws-jsonrpc": "3.3.2",
```

Comment:
the packages were sorted by name. can we put those two back to order?

Reply:
Moved.